dev-db/influxdb/metadata.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="person">
+    <email>gaetan@nexylan.com</email>
+    <name>Gaetan Allart</name>
+  </maintainer>
+  <longdescription>
+    InfluxDB is a modern Open-Source times-series database platform
```

Comment:
If it's open source, then why are you installing a binary Debian package instead of building it?